packages/core/src/components/tooltip/_tooltip.scss

```diff
@@ -30,15 +33,51 @@
     padding: $tooltip-padding-vertical $tooltip-padding-horizontal;
   }
 
+  // override typography because we use a dark background
```

Comment:
I'm planning to implement a more comprehensive typography refactor here to make all the text styles work better inside tooltips... coming in a separate PR